cmd/docgen/run.go

```diff
@@ -71,19 +74,20 @@ var contactDef = `
 
 func createExampleSession(assetsDef string) (flows.Session, error) {
 	// read our assets
-	assets := engine.NewAssetStore()
-	if err := assets.IncludeAssets(json.RawMessage(assetsDef)); err != nil {
+	assetCache := engine.NewAssetCache()
```

Comment:
`AssetCache` isn't ideal name as it's also responsible for fetching assets from the caller.. but we thus far failed to come up with anything better

Reply:
`AssetStore`?

Reply:
I've gone back and forth on AssetCache vs AssetStore and @ericnewcomer said he preferred the former.